src/reistijden_v1/tests.py

```diff
@@ -0,0 +1,383 @@
+            </measurementSiteReference>
+            <trafficFlowData>
+                <measuredFlow specificLane="lane1">
+                    <vehicleFlow>6</vehicleFlow>
+                    <numberOfInputValuesUsed>
+                        <category count="6" type="Auto" />
+                    </numberOfInputValuesUsed>
+                </measuredFlow>
+            </trafficFlowData>
+        </siteMeasurements>
+        <siteMeasurements>
+            <measurementSiteReference id="22" version="1.0">
+                <measurementSiteType>location</measurementSiteType>
+                <location>
+                    <lane specificLane="lane1">
+                        <camera id="222">
+                            <coordinates latitude="52.222222" longitude="4.222222" />
+                            <laneNumber>1</laneNumber>
+                            <status>on</status>
+                            <viewDirection>222</viewDirection>
+                        </camera>
+                    </lane>
+                </location>
+            </measurementSiteReference>
+            <trafficFlowData>
+                <measuredFlow specificLane="lane1">
+                    <vehicleFlow>6</vehicleFlow>
+                    <numberOfInputValuesUsed>
+                        <category count="6" type="Auto" />
+                    </numberOfInputValuesUsed>
+                </measuredFlow>
+            </trafficFlowData>
+        </siteMeasurements>
+        <siteMeasurements>
+            <measurementSiteReference id="33" version="1.0">
+                <measurementSiteType>location</measurementSiteType>
+                <location>
+                    <lane specificLane="lane1">
+                        <camera id="333">
+                            <coordinates latitude="52.333333" longitude="4.333333" />
+                            <laneNumber>1</laneNumber>
+                            <status>on</status>
+                            <viewDirection>333</viewDirection>
+                        </camera>
+                    </lane>
+                </location>
+            </measurementSiteReference>
+            <trafficFlowData>
+                <measuredFlow specificLane="lane1">
+                    <vehicleFlow>6</vehicleFlow>
+                    <numberOfInputValuesUsed>
+                        <category count="6" type="Auto" />
+                    </numberOfInputValuesUsed>
+                </measuredFlow>
+            </trafficFlowData>
+        </siteMeasurements>
+    </payloadPublication>
+</amsterdamTravelTimes>
+"""
+
+AUTHORIZATION_HEADER = {'HTTP_AUTHORIZATION': f"Token {settings.AUTHORIZATION_TOKEN}"}
+
+
+class ReistijdenPostTest(APITestCase):
+    def setUp(self):
+        self.URL = '/reistijden/v1/'
+
+    def test_post_new_travel_time(self):
+        """ Test posting a new vanilla travel time message """
+        response = self.client.post(self.URL, TEST_POST_TRAVEL_TIME, **AUTHORIZATION_HEADER, format='xml')
```

Comment:
Je moet nu steeds deze regel herhalen met kleine aanpassingen. Ik zou een generieke method maken:
```
def execute_request(request_func, url, data=None, format='xml'):
    return request_func(url=url, data=data, **AUTHORIZATION_HEADER, format=format)
```

Daarna kan je die zo gebruiken:
```
response = self.execute_request(self.client.post, self.URL, TEST_POST_TRAVEL_TIME)
response = self.execute_request(self.client.delete, self.URL, format='json')
```

Reply:
Als ik het zo bekijk win je er niet extreem veel mee, maar dan hoef je in ieder geval niet die AUTH HEADER steeds te herhalen. Een echt groot probleem lost het echter niet op

Reply:
Zou idd ook kunnen. Maar je wint er idd weinig karakters mee, en het maakt het er niet veel leesbaarder op. Voor nu laat ik het dus zo.